packages/info/index.ts

```diff
@@ -1,11 +1,12 @@
 import * as envinfo from "envinfo";
+import * as process from "process";
 
 /**
  * Prints debugging information for webpack issue reporting
  */
 
 export default async function info() {
-	console.log(
+	process.stdout.write(
```

Comment:
is this still throwing console error with the rule enabled?

Reply:
No, it would show the warning. We can decide on one standard convention to follow between `console` or `process`.